src/components/DatePicker/DatePicker.vue

```diff
@@ -114,6 +115,11 @@ export default class DatePicker extends Vue {
     this.dateText = this.initialValue
   }
 
+  /** The display Date. */
+  private get displayDate (): string {
+    return this.utcDateToDisplayDate(new Date(this.dateText))
```

Comment:
Due to date-PTSD, I now cringe when I see `new Date(...)`. Example:

```
> new Date('2021-11-29').toUTCString()
> 'Mon, 29 Nov 2021 00:00:00 GMT'

> new Date('2021-11-29').toISOString()
> '2021-11-29T00:00:00.000Z'

> new Date('2021-11-29').toString()
> 'Sun Nov 28 2021 16:00:00 GMT-0800 (Pacific Standard Time)'

> new Date('2021-11-29').toDateString()
> 'Sun Nov 28 2021'
```

It _really_ depends what you do with the resulting Date object. For example, if you just chop off the time portion, you can see that the date (in Pacific tz) is yesterday. (A Date object is always UTC internally, but it may display in "local" timezone as above.)

Passing this to `dateToPacific()` should work fine, since it explicitly converts it to Pacific timezone.

We use `new Date(...)` more than I'd like in our code. I'm sure some of those are incorrect since they intend to convert a "local" time string to a Date object. Unfortunately "local" depends on the browser's timezone and date and time... which could easily not be Pacific, or could be wrong. That is why I wrote `createUtcDate()` -- it explicitly converts from Pacific to a (UTC) Date object.

Reply:
Ok right on, i will give this a try. 
When i was dealing with Pacific time prior, the date selected in the calendar and the display date were off by the offset. 
Will try what you suggest above!

Reply:
I believe your current code is fine. This was just a (long-winded) FYI.

Reply:
no no i agree, Dates are a thorn. If we can simplify this, definitely will. 
Having a look 👁️

Reply:
Except... call dateToPacificDate instead of utcDateToDisplayDate.

Reply:
I think we have the date conversion methods we need, now.

(Unfortunately, I don't we have a date mixin in any of our UIs that contains all of them, sigh. Future work.)

Reply:
Was able to do away with the added method and utlize `yyyyMmDdToPacificDate` that currently exists. 
Since we are strictly just converting the format for a display purpose. 
This also does away with creating a new Date object.
Much cleaner!

Reply:
You'll need to add `yyyyMmDdToPacificDate` to the local date mixin in the shared library.

Reply:
Updated!